packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1155,62 +1157,136 @@ You can apply the following conditions to the Rules associated with a global ses
 
-> **Note:** The MFA Policy API is a <ApiLifecycle access="beta" /> release.
+<ApiLifecycle access="ie" />
 
-The Multifactor (MFA) Enrollment Policy controls which MFA methods are available for a User, as well as when a User may enroll in a particular Factor.
+> **Note:** In Identity Engine, the Multifactor (MFA) Enrollment Policy name has changed to authenticator enrollment policy. The policy type of `MFA_ENROLL` remains unchanged, however, the `settings` data is updated for authenticators. For Classic Engine, see [Multifactor (MFA) Enrollment Policy](#multifactor-mfa-enrollment-policy).
+> The authenticator enrollment policy is a <ApiLifecycle access="beta" /> release.
 
-#### Policy Factors Settings example
+The authenticator enrollment policy controls which authenticators are available for a User, as well as when a User may enroll in a particular authenticator.
 
-> **Note:** Policy Settings are included only for those Factors that are enabled.
+#### Authenticator enrollment policy settings example
+
+<ApiLifecycle access="ie" />
+
+> **Note:** Policy settings are included only for those authenticators that are enabled.
 
 ```json
    "settings": {
-     "factors": {
-       "okta_question": {
+     "type": "AUTHENTICATORS",
+     "authenticators": [
+       {
+         "key": "security_question",
          "enroll": {
            "self": "OPTIONAL"
-         },
-         "consent": {
-           "type": "NONE"
          }
        },
-       "okta_sms": {
+       {
+         "key": "okta_phone",
          "enroll": {
-           "self": "REQUIRED"
-         },
-         "consent": {
-           "type": "NONE"
+           "self": "OPTIONAL"
          }
        }
-     }
+     ]
    }
 ```
 
-#### Policy Authenticators Settings example
+### Policy Settings data
+
+<ApiLifecycle access="ie" />
+
+| Parameter                                                                        | Description                           | Data Type                                                                    | Required | Default   |
+| ---                                                                              | ---                                   | ---                                                                          | ---      | ---       |
+| authenticators | List of authenticator policy settings | Array of [Policy Authenticator object](#policy-authenticator-object) | No       |           |
+| factors                                                                          | Factor policy settings. This parameter is for Classic Engine MFA Enrollment policies that have migrated to Identity Engine but haven't converted to using authenticators yet. Factors and authenticators are mutually exclusive in an authenticator enrollment policy. When a policy is updated to use authenticators, the factors are removed.               | [Policy Factors Configuration object](#policy-factors-configuration-object)  | No       |           |
```

Comment:
Is this table describing the object structure above? If so I don't see "Factors" in the code snippet.

Reply:
`factors` is optional and mutually exclusive to `authenticators`. You will only see one or the other in the `settings` structure.

Reply:
okay